src/util/secrets.ts

```diff
@@ -41,7 +40,7 @@ required.forEach((value: string) => {
 if (process.env.NODE_ENV === 'test') {
     if (process.env.PORT_TEST !== undefined) process.env.PORT = process.env.PORT_TEST;
     if (process.env.MONGODB_URI_TEST !== undefined) process.env.MONGODB_URI = process.env.MONGODB_URI_TEST;
-    if (process.env.TESTNET_RPC_TEST !== undefined) process.env.TESTNET_RPC = process.env.TESTNET_RPC_TEST;
+    if (process.env.MAINNET_RPC_TEST !== undefined) process.env.RPC = process.env.MAINNET_RPC_TEST;
```

Comment:
Shouldn't this be `process.env.RPC_TEST`?

To make it all a bit more clear we could change the suffix to `_TEST_OVERRIDE`. So the whole thing becomes
```
    if (process.env.PORT_TEST_OVERRIDE !== undefined) process.env.PORT = process.env.PORT_TEST_OVERRIDE;
    if (process.env.MONGODB_URI_TEST_OVERRIDE !== undefined) process.env.MONGODB_URI = process.env.MONGODB_URI_TEST_OVERRIDE;
    if (process.env.RPC_TEST_OVERRIDE !== undefined) process.env.RPC = process.env.RPC_TEST_OVERRIDE;
 ```